.github/workflows/test.yml

```diff
@@ -4,11 +4,21 @@ on:
     branches:
     - 'main'
   pull_request:
+  workflow_dispatch:
+    inputs:
+      build_and_test_all_packages:
+        type: boolean
+        description: Build and test all packages
```

Comment:
Not yet implemented but the input is logged so that I can later inspect the possible values and hook up to the internals

Reply:
Has been implemented